GNUmakefile

```diff
@@ -65,16 +65,20 @@ SERVICE ?= gitlab-ce
 GITLAB_TOKEN ?= ACCTEST1234567890123
 GITLAB_BASE_URL ?= http://127.0.0.1:8080/api/v4
 
-testacc-up: ## Launch a GitLab instance.
+testacc-up: | certs ## Launch a GitLab instance.
```

Comment:
It might be better to depend on the files instead of the folder ...

Reply:
So my only thought on leaving as the folder is because the order-only dependency seems to be linked to individual files or individual directories vs a group of files.  So for instance if I were to have `| certs/gitlab-registry.crt certs/gitlab-registry.key` I would have to have 2 makefile targets but underneath them I would still be using the openssl command to regenerate both certs.